### DISCUSSION TITLE
"M20 L" implementation - SD long filenames report

### DISCUSSION THREAD

**Discussion by Graylag-PD** [luc-github/ESP3D#698] [Ideas]
So, I know there have been discussions about 8.3 file name format in Marlin forever and I understand there was not universally accepted solution. But recently released 2.0.9.2 finally added M20 L parameter which should report long filenames and which made it defacto standard. So, is there any plan to support it in ESP3D? It *should* be relatively easy to implement, yet would increase user comfort significantly.
Or is there any significant problem with this?

**Comment by luc-github** [comment]
last status is this one : https://github.com/luc-github/ESP3D-WEBUI/issues/183

**Reply by Graylag-PD** [reply]
Oh, ok, must have missed that discussion. Thanks for answer!

**Reply by luc-github** [reply]
the file support is actually mostly in webUI - the FW is only a bridge with extra features
